MANIFEST.in

```diff
@@ -1,17 +0,0 @@
-include AUTHORS.rst
-include CONTRIBUTING.rst
-include LICENSE
-include README.rst
-include requirements.txt
-
-recursive-exclude * __pycache__
-recursive-exclude * *.py[co]
-
-recursive-include docs *.rst conf.py Makefile make.bat
-
-include versioneer.py
-include event_model/_version.py
-include event_model/schemas/*.json
```

Comment:
Please can you try running the wheel build commands locally, and compare against the last release, especially checking that the schemas are included